static/js/compose_state.js

```diff
@@ -51,13 +52,16 @@ function get_or_set(fieldname, keep_leading_whitespace, no_trim) {
 }
 
 export function stream_name() {
-    return $("#stream_message_recipient_stream").val().trim();
+    if (compose_ui.compose_stream_widget) {
```

Comment:
Does this ever fail?

Reply:
Ah, this is initialized once on pageload and not each time the compose box opens. So it would always be defined, cool. Thanks for asking about that.

Reply:
Removed this check